app/components/check_answers_summary_list_card_component.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="x-govuk-summary-card govuk-!-margin-bottom-6">
+  <% if applicable_questions.first.check_answers_card_number != 0 %>
+    <div class="x-govuk-summary-card__header">
+      <% if applicable_questions.first.check_answers_card_number == 1 %>
+        <h3 class="x-govuk-summary-card__title">Lead tenant</h3>
```

Comment:
This should be coming from the page/question definition shouldn't it? At present this isn't really a re-usable component.